src/main/java/ro/unibuc/triplea/domain/auth/exception/advice/AppWideExceptionHandler.java

```diff
@@ -1,20 +1,29 @@
 package ro.unibuc.triplea.domain.auth.exception.advice;
 
+import org.springframework.core.annotation.AnnotationUtils;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import ro.unibuc.triplea.application.auth.dto.response.StandardResponse;
 import ro.unibuc.triplea.domain.auth.exception.NotFoundException;
 import ro.unibuc.triplea.domain.auth.exception.ValidateException;
+import ro.unibuc.triplea.domain.games.steam.exception.SteamGameNotFoundException;
+import ro.unibuc.triplea.domain.games.steam.exception.SteamGameValidateException;
 
 @RestControllerAdvice
 @CrossOrigin
 public class AppWideExceptionHandler {
-    @ExceptionHandler(Exception.class)
-    public ResponseEntity handleException(Exception e) {
-        return new ResponseEntity(new StandardResponse("500", "Error", e.getMessage()), HttpStatus.INTERNAL_SERVER_ERROR);
+    @ExceptionHandler(SteamGameNotFoundException.class)
+    public ResponseEntity<StandardResponse> handleNotFoundException(SteamGameNotFoundException e) {
+        return new ResponseEntity<StandardResponse>(new StandardResponse("404", "Error", e.getMessage()), HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(SteamGameValidateException.class)
+    public ResponseEntity<StandardResponse> handleValidationException(SteamGameValidateException e) {
```

Comment:
This violates SRP. Move Games to it's own controller advice.

Reply:
Fixed. Maybe a more specific name for this handler can be given, not ```AppWideExceptionHandler```.